app/javascript/packs/application/components/__snapshots__/TourContainer.test.js.snap

```diff
@@ -5,6 +5,11 @@ exports[`<TourContainer /> render 1`] = `
   <Connect(Tour)
     tourId="1"
   />
+  <[object Object]
```

Comment:
Why does the link render as, presumably, `Link.toString()` in the snapshot whereas `Connect` renders appropriately?  This is more of a general question for my education because I don't have much experience using these sorts of snapshots.

Reply:
Good catch! I didn't notice that before.
I did research and found that it may have something to do with the shallow render from enzyme, since that does not render child components or it could have something to do with reach-router, which is where I'm getting the Link component. However, I found zero solutions that worked so I'll ask Ray about it and get back to you with what I find!

Reply:
In my experience Link has needed more of the infrastructure to be running than we typically want to run in a component test.  Sometimes it's necessary to get that stuff up and running, but usually shallow rendering is fine.  I think this is probably desired behavior.